src/contracts/AbstractARM.sol

```diff
@@ -112,7 +114,14 @@ abstract contract AbstractARM is OwnableOperable, ERC20Upgradeable {
     /// @notice The address of the CapManager contract used to manage the ARM's liquidity provider and total assets caps.
     address public capManager;
 
-    uint256[41] private _gap;
+    /// @notice The address of the active lending market.
+    address public activeMarket;
```

Comment:
Code might be prettier if `activeMarket` were typed as a IERC4626, instead of needing to be casted to an IERC4626 when it's used.

Reply:
It's either declare the storage variable as `IERC4626` can cast to `address` when needed. eg param validation or token approve. Or declare as `address` and cast to `IERC4626`

activeMarket  is used as `IERC4626` nine times and `address` eight times.

I prefer to cast to `IERC4626` when needed as it makes it clear the functions belong to `IERC4626`

Reply:
![image](https://github.com/user-attachments/assets/fadff1d3-3804-40e7-aee8-49f78154aac7)

I am fine with that.